src/components/ui/button.tsx

```diff
@@ -16,7 +20,7 @@ const buttonVariants = cva(
         outline:
           "border border-gray-200 text-gray-900 bg-white shadow-sm hover:bg-gray-100 hover:text-gray-900 dark:border-gray-800 dark:bg-gray-950 dark:hover:bg-gray-800 dark:hover:text-gray-50",
         primary:
-          "bg-primary-700 text-white shadow hover:bg-primary-700/90 dark:bg-primary-100 dark:text-primary-900 dark:hover:bg-primary-100/90",
+          "bg-primary-500 text-white shadow hover:bg-primary-500/90 dark:bg-primary-100 dark:text-primary-900 dark:hover:bg-primary-100/90",
```

Comment:
let's stick with whats as per new designs 

```suggestion
          "bg-primary-700 text-white shadow hover:bg-primary-700/90 dark:bg-primary-100 dark:text-primary-900 dark:hover:bg-primary-100/90",
```